### DISCUSSION TITLE
Introduce Yourself

### DISCUSSION THREAD

**Discussion by scotthibbs** [w7sst/MorseRunner#65] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Discuss design ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by scotthibbs** [comment]
I moved Mike's prior entries to here so we could start this Introduce yourself discussion. 

My name is Scott Hibbs KD4SIR. I live in Evansville Indiana. I got involved with GitHub through my other project "FDLog Enhanced". I liked that program so much that I started to learn Python just so I could enhance it. I've been a ham for over 30 years and I currently enjoy 40M FT8 and OMISS nets. I use Morse Runner to pretend that I know Morse Code. 

I created a welcome to Morse Runner Community Edition video on my YouTube page. It can be found at https://youtu.be/Y1kHqJRytMM

I really hope you join us - and please take a moment to introduce yourself. 

Since I don't program Pascal, I'll focus on documentation, program testing and the GUI. Oh, I'll also cheerlead if you need it. :)

Scott, KD4SIR

**Comment by w7sst** [comment]
My name is Mike Brashler, W7SST. Early last spring, I got the idea of adding the ARRL Field Day contest to Morse Runner. I started researching the various forks from VE3NEA's original work and discovered many independent development efforts. Since then, I have implemented a generalized, table-driven approach to supporting multiple contests within Morse Runner, including CQWPX, CWOPS CWT, ARRL Field Day, and NCL NAQP.

My long-term goal is to combine the various development efforts into a common repository and encourage other developers and users to join in this effort. Hopefully we can get a group of people working on a common code base. Over the next year, I plan to pull changes/enhancements from the various development branches into this repository.

I invite you to:
- Visit the [Morse Runner Community Edition repository](https://github.com/w7sst/MorseRunner#readme) to learn more about the project and goals.
- Please consider joining this effort as a Contributor and help as an Maintainer, Contributor, Community Member, or project advisor.
- If you want to install and run the multi-contest MR prototype, it is available [here](https://github.com/w7sst/MorseRunner/releases).

Later this fall, we will announce this project to a larger community (e.g N1MM user community) to gain additional resources. Scott Hibbs, KD4SIR, has plans to announce this project to his local ham radio club and post a YouTube video. For now, I'd like to form a core group of contributors to get this started.

Please let me know if you have any feedback or if I can answer any questions.

Thank you and best regards.

73,
Mike Brashler, W7SST

**Comment by jr8ppg** [comment]
My name is Takayoshi Muto, JR8PPG.
Thank you for inviting me to this project.
I live in Sapporo, Japan.
I customized MorseRunner for the domestic contest in Japan.
Because the numbers are very long and good for practice.
My first goal is to port my changes to MorseRunner CE.
Thank you and best regards.

73
Taka, JR8PPG

**Reply by w7sst** [reply]
Taka, Welcome to the group! We look forward to your contributions. Please let me know if you have any questions as you proceed.

Keep in mind that our `main` branch is considered a prototype and will change over time as a better design emerges. You should have not problem integrating your work into the existing framework. We can evolve the design over time. We can start a discussion in #82 discussion topic on how to approach this project and to answer specific questions.

Best regards,
Mike, W7SST

**Comment by BH1SCW** [comment]
hello hello ,  I am Fanjun Kong, BH1SCW
My homepage is  https://www.bh1scw.com
and here:  https://www.qrz.com/db/BH1SCW

I am so happy that mr is still developing. 
Thanks so much

**Comment by tekenny** [comment]
hi all, Tom NJ2DX here. Professional software developer for decades and wondering how I can help…

**Reply by w7sst** [reply]
Hi Tom, welcome aboard. Glad to have your assistance to this project. I see you have already grabbed an Issue to work on - sounds good.

Feel free to propose changes to any of the supporting material on this github site. You have the advantage of being a newcomer, so you can give us a review as a new developer to the project. If you see any holes, let me know or try to add additional information as you see fit. I would also read through some of the closed Issues and pull requests to get a flavor for the work being done. I'm current working on refactoring and adding the ARRL DX Contest. Good to have you with us.

73, Mike W7SST

**Reply by tekenny** [reply]
Mike, thanks. I think I'll write up new developer getting started docs to help future contributors onboard. 73 Tom NJ2DX

**Comment by w7sst** [comment]
Awesome! Thank you.

**Comment by K6OK** [comment]
I'm a ham in California, a retired civil engineer, although on the side I do structural analysis of towers to help ham get permits. I'm not a professional developer.  I've learned enough Python and QT to be dangerous.  I have a remote station where I do my station control with small Linux boards using Python and QT5.  A long time ago I did a little bit of work in Delphi but I've forgotten most of it.  From what I've seen making the jump from Python to Pascal doesn't look too intimidating.

In the recent 2022 CQWWCW I made 1,200 QSOs.  I did drills daily before the contest with 1.81 and it really helped me with my ability to run stations.

73 Jim K6OK
